java/src/org/openqa/selenium/Rectangle.java

```diff
@@ -64,6 +65,11 @@ public Dimension getDimension() {
     return new Dimension(width, height);
   }
 
+  private Map<String, Object> toJson() {
+    return Map.of("class", getClass().getName(),
```

Comment:
Thank you for your contribution. I wonder if adding "class" is required. I have not seen this in other toJson() methods.

Reply:
It's not strictly necessary, but the defaults serialization performed by **JsonOutput** adds this entry to the map. I'll take it out.